core/templates/dev/head/pages/exploration_player/AnswerClassificationService.js

```diff
@@ -43,6 +54,7 @@ oppia.factory('AnswerClassificationService', [
     var classifyAnswer = function(
         answer, answerGroups, defaultOutcome, interactionRulesService) {
       // Find the first group that contains a rule which returns true
+      // TODO(bhenning): Implement soft and classifier classifications.
```

Comment:
We are doing soft classification, though I updated this comment to refer to it correctly. Since the original rule migration to the frontend removed training data classification, I see no need to do this before the answer migration. I'll file an issue to track this, though. It should be a fairly easy change. #3287